Modules/Core/Tests/Unit/MailerHelperTest.php

```
@@ -192,4 +192,129 @@ public function it_validates_email_with_new_tlds(): void
+        );
+
+        $reflection = new \ReflectionMethod(MailerHelper::class, 'email_invoice');
+        $this->assertTrue($reflection->isStatic(), 'email_invoice should be a static method');
+        $this->assertTrue($reflection->isPublic(), 'email_invoice should be public');
+        
+        $parameters = $reflection->getParameters();
+        $this->assertCount(9, $parameters, 'email_invoice should have 9 parameters');
+        $this->assertEquals('invoice_id', $parameters[0]->getName());
+        $this->assertEquals('invoice_template', $parameters[1]->getName());
+        $this->assertEquals('from', $parameters[2]->getName());
+    }
```

PHPMD (2.15.0)

The method it_has_email_invoice_method is not named in camelCase. (undefined)

(CamelCaseMethodName)